src/classes/STG_PanelOppNaming_CTRL.cls

```diff
@@ -183,21 +206,92 @@ public with sharing class STG_PanelOppNaming_CTRL extends STG_Panel {
     }
 
     /*********************************************************************************************************
-    * @description The list of SelectOptions for Opportunity Attribution 
+    * @description The list of SelectOptions for Opportunity Attribution
     */
-    static public list<SelectOption> listSOOpportunityAttribution {
-        get {    
+    public static list<SelectOption> listSOOpportunityAttribution {
```

Comment:
Verify the NPSP Settings > Donations > Opportunity Names page for both view and edit works when there is already a custom settings contains "Contact Donations" value.

Reply:
> Verify the NPSP Settings > Donations > Opportunity Names page for both view and edit works when there is already custom settings containing "Contact Donations" value.

I am trying to verify this for existing org with 'Contact Donations' value. Not sure if install script changes sets custom settings value in scratch org, I thought it will work only in production org.

Reply:
@RadSFDeveloper , it works after running the script in existing org with "Contact Donations" value.